components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -1233,8 +1233,7 @@ def enrolled_with(enrollment)
       EVIDENCES.each do |evidence_type|
         evidence = self.send(evidence_type)
         next unless evidence.present?
-        enrollment_member = enrollment.hbx_enrollment_members.where(:applicant_id => family_member_id).first
-        aptc_or_csr_used = enrollment_member.applied_aptc_amount > 0 || ['03', '04', '05', '06'].include?(enrollment.product.csr_variant_id)
+        aptc_or_csr_used = enrollment.applied_aptc_amount > 0 || ['03', '04', '05', '06'].include?(enrollment.product.csr_variant_id)
```

Comment:
Why are we replacing `enrollment_member` with `enrollment` here ?

Reply:
We want to check the aptc/csr on the enrollment rather than the enrollment member because if a user chooses a plan without aptc/csr but they are eligible for it, the code logic would update their evidence as if aptc/csr was applied to the plan, providing errors. Checking the aptc/csr from the enrollment directly allows the logic to check if aptc/csr was actually applied